CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+### Before you Start
+- Unless stated in README.md. Make sure you are pulling the source code from main branch with recursive variable to pull all submodules Collapse need to run.
+
+    ```git pull https://github.com/neon-nyan/Collapse --recursive```
+    
+- Make sure you use the supported IDE/SDKs listed below.
+- Make sure to read the Feature(s) restriction below.
+- When its time to do a Pull Request, do a pull request to the same branch when you first pulled.
+- We don't require a fully descriptive PR description, but please be clear on what added/removed and what it does.
+
+### Software Needed
+Below is the software needed to develop and contributing to this project. Also, Collapse uses many submodule inside its repository, so don't forget to use recursive variable when pulling this repository to your local computer.

```

Comment:
- Below is the software needed -> Below are a list of tools required
- Remove develop, since it's implied through "contributing"
- contributing -> contribute
- Remove "Also", "inside"
- so don't forget to use recursive variable when pulling this repository to your local computer -> so don't forget to use the `--recurse` parameter when cloning this repository to your local machine.

Final sentence: Below are a list of tools required to develop & contribute to this project. Since Collapse uses many submodules in its repository, don't forget to use the `--recurse` parameter when cloning this repository to your local machine to ensure that you have a working development environment.

Reply:
deleted the entire `Also, Collapse uses many submodule inside its repository, so don't forget to use recursive variable when pulling this repository to your local computer.` section as its redundant from ##Before you Start section. Applied other changes